pbin.sh

```diff
@@ -22,10 +22,14 @@
 
 set -e
 PROGRAM=${0##*/}
+verbose=""
+result=""
 
-# can be overriden from env
-: ${PASTE_URL='http://paste.mate-desktop.org/api/create'}
-#: ${PASTE_APIKEY='apikey'}
+if [ -z "$PASTE_URL" ];then
```

Comment:
unwanted chunk. the previous code did exactly the same, so the -z test is superfluous.